commons/all/src/main/java/org/infinispan/commons/netty/ReplayableByteBuf.java

```diff
@@ -0,0 +1,8 @@
+package org.infinispan.commons.netty;
+
+import io.netty.buffer.ByteBuf;
+
+public interface ReplayableByteBuf {
```

Comment:
Nasty trick just to be able to read the value without throwing the replay signal :-1: